cpu/atmega328p/Makefile.include

```diff
@@ -0,0 +1,24 @@
+# this CPU implementation is using the new core/CPU interface
+export CFLAGS += -DCOREIF_NG=1
+
+# tell the build system that the CPU depends on the atmega common files
+USEMODULE += atmega_common
+
+# export the peripheral drivers to be linked into the final binary
+export USEMODULE += periph
+
+# the atmel port uses uart_stdio
+export USEMODULE += uart_stdio
+
+# define path to atmega common module, which is needed for this CPU
+export ATMEGA_COMMON = $(RIOTCPU)/atmega_common/
+
+# define the linker script to use for this CPU
+#export LINKERSCRIPT = $(RIOTCPU)/$(CPU)/ldscripts/atmega2560.ld
```

Comment:
I don't think so, thats why I've commented this line :-)
Better remove it completely.
fixed